### PR TITLE
--no-cache-dir option added

### LINKED ISSUE #231: Always use --no-cache-dir option for pip
Whenever we are using pip in .taskcluster.yml and in docker files, we should use the options `--disable-pip-version-check`, `--no-cache-dir` and `--quiet`.

### PR DESCRIPTION
**Pull request by Patil2099**
Fixes #231

### CODE REVIEW

**Comment by marco-c** [comment]
Thanks! The Docker files need to be updated too.

**Comment by Patil2099** [comment]
@marco-c is this correct ?

File: backend/Dockerfile [hunk 1]
```diff
@@ -3,7 +3,7 @@ FROM python:3.7-slim
 ADD tools /src/tools
 ADD backend /src/backend
 
-RUN cd /src/tools && python setup.py install
-RUN cd /src/backend && python setup.py install
+RUN cd /src/tools && python --disable-pip-version-check --no-cache-dir --quiet  setup.py install
```
Comment (La0) [inline backend/Dockerfile]: This cannot work as you added pip arguments to a classic `python setup.py install` script. Those are not supported.

You can convert the `python setup.py install` to `pip install .` and add the arguments.

**Comment by La0** [comment]
Also, please try to build the docker images on your computer (with `docker build backend/Dockerfile .` for example) before pushing. You would have noticed the errors that way.

File: bot/Dockerfile [hunk 2]
```diff
@@ -5,7 +5,7 @@ ADD bot /src/bot
 
 RUN /src/bot/ci/bootstrap.sh
 
-RUN cd /src/tools && python setup.py install
-RUN cd /src/bot && python setup.py install
+RUN cd /src/tools && pip install --disable-pip-version-check --no-cache-dir --quiet  . 
+RUN cd /src/bot && pip install--disable-pip-version-check --no-cache-dir --quiet  .
```
Comment (La0) [inline bot/Dockerfile]: Missing space after `install`.

Please try building locally before pushing.

File: bot/Dockerfile [hunk 3]
```diff
@@ -5,7 +5,7 @@ ADD bot /src/bot
 
 RUN /src/bot/ci/bootstrap.sh
 
-RUN cd /src/tools && python setup.py install
-RUN cd /src/bot && python setup.py install
+RUN cd /src/tools && pip install --disable-pip-version-check --no-cache-dir --quiet .
```
Comment (La0) [inline bot/Dockerfile]: Please remove the trailing whitespace

```suggestion
RUN cd /src/tools && pip install --disable-pip-version-check --no-cache-dir --quiet .
```

File: backend/Dockerfile [hunk 4]
```diff
@@ -3,7 +3,7 @@ FROM python:3.7-slim
 ADD tools /src/tools
 ADD backend /src/backend
 
-RUN cd /src/tools && python setup.py install
-RUN cd /src/backend && python setup.py install
+RUN cd /src/tools && pip install --disable-pip-version-check --no-cache-dir --quiet .
+RUN cd /src/backend && pip install --disable-pip-version-check --no-cache-dir --quiet .
```
Comment (La0) [inline backend/Dockerfile]: Please remove the trailing whitespace

```suggestion
RUN cd /src/backend && pip install --disable-pip-version-check --no-cache-dir --quiet .
```

File: events/Dockerfile [hunk 5]
```diff
@@ -4,7 +4,7 @@ FROM python:3.7-slim
 ADD tools /src/tools
 ADD events /src/events
 
-RUN cd /src/tools && python setup.py install
-RUN cd /src/events && python setup.py install
+RUN cd /src/tools && pip install --disable-pip-version-check --no-cache-dir --quiet .
```
Comment (La0) [inline events/Dockerfile]: Please remove the trailing whitespace

```suggestion
RUN cd /src/tools && pip install --disable-pip-version-check --no-cache-dir --quiet .
```

File: events/Dockerfile [hunk 6]
```diff
@@ -4,7 +4,7 @@ FROM python:3.7-slim
...
+RUN cd /src/events && pip install --disable-pip-version-check --no-cache-dir --quiet .
```
Comment (La0) [inline events/Dockerfile]: One more

```suggestion
RUN cd /src/events && pip install --disable-pip-version-check --no-cache-dir --quiet .
```

**Comment by Patil2099** [comment]
@La0 I am really sorry about the inconvenience caused.

**Pull request merged**